orchagent/fdborch.cpp

```diff
@@ -417,13 +417,48 @@ void FdbOrch::doTask(NotificationConsumer& consumer)
     }
 }
 
+void FdbOrch::flushFDBEntry(sai_object_id_t bridge_port_oid,
```

Comment:
Should be flushFDBEntries() or flushFdbEntries()

Reply:
For a given <bvid, bridge_port_oid>, can we have more than 1 FDB entry?

Reply:
yes, for VMs or any scenario of multiple MACs behind the port+vlan.

Reply:
Renamed the function!